tests/components/weheat/const.py

```diff
@@ -4,3 +4,10 @@
 CLIENT_SECRET = "5678"
 
 USER_UUID_1 = "0000-1111-2222-3333"
+
+APPLICATION_CREDENTIALS = "application_credentials"
+
+CONF_REFRESH_TOKEN = "refresh_token"
+CONF_AUTH_IMPLEMENTATION = "auth_implementation"
```

Comment:
Aren't these in homeassistant.const?

Reply:
no. "CONF_REFRESH_TOKEN" is defined in many components so could make a great addition to homeassistant.const